From 25639f527db8aba8e54e606b87495fcf26ad99f9 Mon Sep 17 00:00:00 2001
From: Theo GOUREAU <theo.goureau-ext@st.com>
Date: Fri, 14 Feb 2025 10:13:23 +0100
Subject: [PATCH 801/841] feat(stm32mp2): add CPU compatible define

Add CPU compatible define for STM32MP2xx family.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git b3541bdc67f87c6e50cd2bd2091341e57bcfbfe6 feat(stm32mp2): add CPU compatible define]
Change-Id: Ieeb390b109f46280fa0865ef1cbe13d5d3eab60d
Signed-off-by: Theo GOUREAU <theo.goureau-ext@st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/437274
Reviewed-by: Maxime MERE <maxime.mere@foss.st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 plat/st/stm32mp2/stm32mp2_def.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/plat/st/stm32mp2/stm32mp2_def.h b/plat/st/stm32mp2/stm32mp2_def.h
index 3c9544c7e..b88e205b6 100644
--- a/plat/st/stm32mp2/stm32mp2_def.h
+++ b/plat/st/stm32mp2/stm32mp2_def.h
@@ -749,5 +749,6 @@ enum ddr_type {
 #define DT_SDMMC2_COMPAT			"st,stm32mp25-sdmmc2"
 #define DT_TAMP_NVRAM_COMPAT			"st,stm32mp25-tamp-nvram"
 #define DT_UART_COMPAT				"st,stm32h7-uart"
+#define DT_CPU_COMPAT				"arm,cortex-a35"
 
 #endif /* STM32MP2_DEF_H */
-- 
2.47.2

